BaseFilter/sections/specific.txt

```diff
@@ -343,6 +343,8 @@ $xmlhttprequest,third-party,denyallow=linkvertise.com|fastly.net|statically.io|s
 !
 ! NOTE: Specific rules
 !
+mmorpg.com##.div-84937-desktop
```

Comment:
There are several such ad leftovers out there, this rule only hides one of them.